tests/test_envs.py

```diff
@@ -3,6 +3,7 @@
 
 # Import for side-effect of registering environment
 import imitation.examples.airl_envs  # noqa: F401
+import imitation.examples.env_suite  # noqa: F401
```

Comment:
@shwang it seems like we're collecting a number of custom environments, how do you think we should structure them? Perhaps have a single import that registers all of them, but have a hierarchy underneath. Gym should only actually import the entry point when we try and make the environment.

Reply:
A single import like `import imitation.examples` registering all the environments sounds good. Also, we should maybe rename `examples` to `envs` (or `examples.envs`) sometime.